pkgs/development/libraries/sofia-sip/default.nix

```diff
@@ -1,19 +1,22 @@
-{ stdenv, fetchurl, glib, openssl, pkgconfig }:
+{ stdenv, fetchFromGitHub, glib, openssl, pkgconfig, autoreconfHook }:
 
 stdenv.mkDerivation rec {
-  name = "sofia-sip-1.12.11";
+  pname = "sofia-sip";
+  version = "1.13";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/sofia-sip/${name}.tar.gz";
-    sha256 = "10bwsdfijpbk9ahlfpk94kzdapxiahl9mljpgwghvq1630pbq09b";
+  src = fetchFromGitHub {
+    owner = "freeswitch";
+    repo = pname;
+    rev = "f6f29b483e9c31ce8d3e87419ec3deea8679312d"; # upstream does not seem to believe in tags
```

Comment:
Maybe add a comment with the date of the commit so it is easier to check it is in a real upstream branch?

Reply:
Better, open an issue upstream asking for tags. I've usually got positive responses from this.

Reply:
I think timestamp is good enough for _this_ release, and hopefully an upstream issue will resolve by the next release.

Reply:
I opened issues upstream but haven't heard anything back in the last two weeks. Should I link them in the derivation instead of the current comment?

Reply:
Yeah, code comment sounds good. GitHub discussions feel less and less reliable medium-term…

Reply:
https://github.com/freeswitch/sofia-sip/releases/tag/v1.13.2